src/App.css

```diff
@@ -73,4 +75,18 @@ h1 {
 h3 {
   font-size: 1.3rem; 
   text-align: center;
```

Comment:
I don't _think_ we want h3 centered. And its size should be smaller than h2.
Also, please organize the headers together.